Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -140,14 +140,17 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
          * @param {EventFacade} e
          */
         _selectLanguage: function (e) {
+            // if the content is considered new: allow changing its language from the list of available languages
+            var isChangingLanguageAllowed = this.get('content').isNew();
```

Comment:
It would name it: `isTranslationAllowed`

Reply:
Variable renamed as suggested. Ping @sunpietro.